### PR TITLE
LPS-125270 Update path to search to check if the tag contains the control-label class

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-125270

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 1]
```diff
@@ -394,7 +394,7 @@
 </tr>
 <tr>
 	<td>CAPTCHA_LABEL</td>
-	<td>//div[@class='taglib-captcha']//label[@class='control-label']</td>
+	<td>//div[@class='taglib-captcha']//label[contains(@class,'control-label')]</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:397]: Show de ⚽ ! 👏

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**